src/bthread/task_group.h

```diff
@@ -247,6 +260,16 @@ class TaskGroup {
         return _control->steal_task(tid, &_steal_seed, _steal_offset);
     }
 
+    bool steal_from_others(bthread_t* tid) {
+        return _control->steal_task(tid, &_steal_seed, _steal_offset);
+    }
+
+    bool NoTasks();
+
+    bool wait();
```

Comment:
You need to have some consistency on naming. If you follow our convention, function names start with a capitalized letter. If you follow brpc convention, function names are in lowercase.